Dockerless Guide.md

```diff
@@ -0,0 +1,135 @@
+
+# Requirements
+
+Make sure to have the following installed:
+- [Node.js](https://nodejs.org/en/download/), active LTS (we recommend [installing via nvm](https://nodejs.org/en/download/package-manager/#nvm))
+- [Go](https://golang.org/), v1.14 or higher
+- a user called `xud`
+
+# XUD
+
+## Option 1: Installing latest release via npm
+This is the easiest and fastest way to install xud on a `amd64` machine: 
+```bash
+sudo npm install xud -g --unsafe-perm
+```
+
+## Option 2: Cloning from GitHub
+
+Testers and developers are encouraged to clone the repository from GitHub and install from source:
+
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+npm run compile
+npm run compile:seedutil
+```
+If you are on an architecture that is *not* `amd64`, you'll have to remove `grpc-tools` and potentially others from the `devDependencies` section of [`package.json`](https://github.com/ExchangeUnion/xud/blob/master/package.json).
+
+## Daemonize `xud`
+
+If you want to daemonize `xud`, so that it starts on boot without needing its own terminal, you can do this using `systemd`:
+
+```toml
+[Unit]
+Description=XUD
+
+[Service]
+User=xud
+Group=xud
+Type=simple
+Environment=NODE_ENV=production
+ExecStart=/home/xud/xud/bin/xud --mainnet
+KillMode=process
+KillSignal=SIGINT
+```
+
+# LND (BTC)
+
+Follow the [lnd installation guide](https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#installing-lnd).
+
+## Daemonize `lnd`
+
+If you want to daemonize `lnd`, so that it starts on boot without needing its own terminal, you can do this using `systemd`:
+
+```toml
+[Unit]
+Description=LND
+
+[Service]
+User=xud
+Group=xud
+Type=simple
+ExecStart=/home/xud/lnd/bin/lnd --bitcoin.mainnet
+KillMode=process
+KillSignal=SIGINT
+```
+
+# Connext
+
+Follow the [docs](https://github.com/connext/vector#quick-start), you want the **`node`: vector node + database** stack.
+
+## Daemonize `connext`
+
+If you want to daemonize `connext`, so that it starts on boot without needing its own terminal, you can do this using `systemd`:
+
+```toml
+[Unit]
+Description=Connext
+
+[Service]
+User=xud
+Group=xud
+Type=simple
+Environment="NODE_ENV=production"
+Environment="CONNEXT_NODE_URL=https://connext.boltz.exchange"
+Environment="CONNEXT_ETH_PROVIDER_URL=http://eth.kilrau.com:41007"
+Environment="LEGACY_MODE=true"
+WorkingDirectory=/home/xud/connext/
+ExecStart=node /home/xud/connext/build/src/index.js
+KillMode=process
+KillSignal=SIGINT
+```
+
+# Tor
+
+You can install tor via `sudo apt install tor` on most linux distros nowadays, just make sure [the version is fairly recent](https://github.com/torproject/tor/releases). If not, consult the [tor installation guides](https://2019.www.torproject.org/docs/installguide.html.en). Run `tor --version` to verify the tor process is running.
```

Comment:
`tor --version` does *not* check whether the Tor process is running

Reply:
Fixed